pkg/operation/update.go

```diff
@@ -28,13 +30,21 @@ func ParseUpdateOperation(request []byte, protocol protocol.Protocol) (*batch.Op
 		return nil, err
 	}
 
+	patches := operationData.Patches[0].GetStringValue(patch.PatchesKey)
```

Comment:
why index 0?

Reply:
This will be fixed with #155 - I will change the model at that time. I already added todo in the code and created issue.